test/unit/org/apache/cassandra/db/filter/ColumnFilterTest.java

```diff
@@ -60,17 +91,13 @@ public void testColumnFilterSerialisationRoundTrip() throws Exception
         ColumnFilter columnFilter;
 
         columnFilter = ColumnFilter.all(metadata);
-        testRoundTrip(columnFilter, ColumnFilter.Serializer.maybeUpdateForBackwardCompatility(columnFilter, MessagingService.VERSION_30), metadata, MessagingService.VERSION_30);
```

Comment:
Now only one of the versions of `testRoundTrip` is called by the tests, we could have a single version of the method.